src/st/lib_psp/st_init_lib.c

```diff
@@ -1,7 +1,30 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 #include "../lib/lib.h"
 
-INCLUDE_ASM("st/lib_psp/psp/lib_psp/1B7F0", func_psp_09253D58);
+#include "../pfn_entity_update.h"
+
+// entity table to allocate new entities
+extern PfnEntityUpdate* PfnEntityUpdates;
```

Comment:
This and `EntityUpdates` are already defined in `pfn_entity_update`

Reply:
You are spot on, I have updated this (and also cleaned up similar code in ST0 and WRP PSP)